inner-scripts/switch-space

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: $0 (init or start or stop | -v)
+
+# Purpose: switch to a given space by setting the appropriate kubeconfig
```

Comment:
I do not understand what "setting the appropriate kubeconfig" means. A kubeconfig is a file. We do not not normally talk about "setting" a file. Is this trying to talk about writing to a file? Or setting an environment variable? Or .. ?

Reply:
I removed switch-space.